Antistasi-Ultimate-Community/A3-Antistasi-Ultimate
# Lootcrate ignore unlocked items parameter

## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [x] Enhancement

### What have you changed and why?
Information:
Added a parameter for the loot crate to ignore items that are already unlocked

### Please specify which Issue this PR Resolves.
closes #XXXX

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)

Make sure the loot crate actually ignores most things that are unlocked, can really only be done in an actual game though

### How can the changes be tested?
Steps:

Enable parameter "Loot crate ignore unlocked items", buy loot crate, use as usual. See if any unlocked items end up in the crate (can check rpt for any items that are already unlocked and get ignored after looting)

********************************************************
Notes:
